app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -187,6 +179,32 @@ class AvatarView @JvmOverloads constructor(
         )
     }
 
+    private fun handleDisplayType(
+        avatarDisplayType: AvatarDisplayType,
+        correspondent: Correspondent?,
+        bimi: Bimi?,
+        contacts: MergedContactDictionary,
+    ) {
+        when (avatarDisplayType) {
+            AvatarDisplayType.UNKNOWN_CORRESPONDENT -> loadUnknownUserAvatar()
+            AvatarDisplayType.CUSTOM_AVATAR,
+            AvatarDisplayType.INITIALS -> {
+                loadAvatarUsingDictionary(correspondent!!, contacts)
+                currentCorrespondent = correspondent
+            }
+            AvatarDisplayType.BIMI -> loadBimiAvatar(ApiRoutes.bimi(bimi!!.svgContentUrl!!), correspondent!!)
+        }
```

Comment:
Should we not reset the `currentBimi` to null the same way we reset `currentCorrespondent` to null?

Also would there be a way to never forget about doing it?